exporter/loggingexporter/logging_exporter.go

```diff
@@ -29,15 +30,16 @@ import (
 )
 
 type loggingExporter struct {
-	logLevel         zapcore.Level
 	logger           *zap.Logger
+	logLevel         zapcore.Level
 	logsMarshaler    plog.Marshaler
 	metricsMarshaler pmetric.Marshaler
 	tracesMarshaler  ptrace.Marshaler
 }
 
-func (s *loggingExporter) pushTraces(_ context.Context, td ptrace.Traces) error {
-	s.logger.Info("TracesExporter", zap.Int("#spans", td.SpanCount()))
+func (s *loggingExporter) pushTraces(ctx context.Context, td ptrace.Traces) error {
+	s.logger.Info("TracesExporter", zap.Int("#spans", td.SpanCount()),
```

Comment:
These entries are emitted at `Info` while the other exporters changed in this PR are exporting at `Debug`.  Why the difference?

Reply:
@Aneurysm9  for logging_exporter.go, it has always been Info() and that is rate limited, so I leave it as is. 
as for oltp.go, I don't see any rate limiter there.   I feel like it is too frequent for Info level.   What do you think?

Reply:
All logging exporter output is at info level since #5678